qt/aqt/data/web/js/webview.ts

```diff
@@ -18,3 +18,14 @@ document.addEventListener("keydown", function (evt: KeyboardEvent) {
         evt.preventDefault();
     }
 });
+
+// prevent Qt's default handling of drag & drop
+document.addEventListener("dragover", function (evt: DragEvent) {
+    evt.preventDefault();
+});
+document.addEventListener("dragenter", function (evt: DragEvent) {
+    evt.preventDefault();
+});
+document.addEventListener("drop", function (evt: DragEvent) {
+    evt.preventDefault();
+});
```

Comment:
If a page decides it wants to opt-in to the default drag/drop behaviour (not prevent the default), how would it do it? Isn't it impossible to remove the registered event listeners without a handle to the anonymous function?

Reply:
Yes, I think it's impossible. I wish there was a function to undo any .preventDefault() calls in other event handlers.

Reply:
Yes, I think it's impossible. I wish there was a function to undo any .preventDefault() calls in other event handlers.